docs/demos/byrd/index.html

```diff
@@ -293,11 +292,35 @@ <h4 class="centeredHeader">Numerator</h4>
                     Filter to features where
                     <select class="fancyBorder num" id="topSearch"> </select>
                     <select class="fancyBorder num" id="topSearchType">
-                        <option value="text">contains the exact text</option>
-                        <option value="rank"
-                            >contains the exact separated text
-                            fragment(s)</option
-                        >
+                        <optgroup label="Text searching">
+                            <option value="text"
+                                >contains the exact text</option
+                            >
+                            <option value="rank"
+                                >contains the exact separated text
+                                fragment(s)</option
+                            >
+                        </optgroup>
+                        <optgroup label="Numeric searching (basic)">
```

Comment:
Is the numeric searching functional in this version?

Reply:
Yep! If you clone the PR (or just clone my fork of Qurro), you can try this out for yourself in the demos.